From f97da84ac1a26b9b8abd41dfd665f7ab2feea99e Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Mon, 14 Aug 2023 22:08:33 +0530
Subject: [PATCH] Correct version number in about dialogue

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 07868861..7f6009e8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([liferea],[1.15.0],[liferea-devel@lists.sourceforge.net])
+AC_INIT([liferea],[1.15.1],[liferea-devel@lists.sourceforge.net])
 AC_CANONICAL_HOST
 AC_CONFIG_SRCDIR([src/feedlist.c])
 
-- 
2.41.0

