From aa1a9d6484db79245268a0fba37d92ec7c353b72 Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Sat, 16 Sep 2023 07:42:57 +0530
Subject: [PATCH] Appdata changes for 1.15.3

---
 net.sourceforge.liferea.appdata.xml.in | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/net.sourceforge.liferea.appdata.xml.in b/net.sourceforge.liferea.appdata.xml.in
index 9e60a685..4b150962 100644
--- a/net.sourceforge.liferea.appdata.xml.in
+++ b/net.sourceforge.liferea.appdata.xml.in
@@ -101,6 +101,16 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release date="2023-09-16" version="1.15.3">
+      <description>
+        <p>This is a new bugfix release for 1.15</p>
+        <ul>Changes
+          <li>Fixes #1297: Enabling plugin 'webkit-settings' crashes Liferea (reported by Paul Gevers)</li>
+          <li>Fixes #1294: Clicking 'remove item' caused a SIGSEGV (reported by Rich Coe)</li>
+          <li>Fixes #1276: Old WebKitCache cache not cleaned (reported by hasezoey)</li>
+        </ul>
+      </description>
+    </release>
     <release date="2023-08-30" version="1.15.2">
       <description>
         <p>This is a bugfix release. It provides an important stability fix regarding feed parsing. Kudos to Rich Coe for debugging and fixing the issue!</p>
--
2.41.0

