--- liferea/plugins/trayicon.py	2023-10-23 12:16:38.771381887 +0530
+++ liferea/plugins/trayicon.py	2023-10-23 12:13:13.126260219 +0530
@@ -157,20 +159,20 @@
         if self.use_appindicator:
             self.indicator = AppIndicator.Indicator.new(
                 "Liferea",
-                Liferea.icon_find_pixmap_file("emblem-web.svg"),
+                "net.sourceforge.liferea",
                 AppIndicator.IndicatorCategory.APPLICATION_STATUS
             )
 
             self.indicator.set_attention_icon_full(
-                Liferea.icon_find_pixmap_file("unread.png"),
+                "net.sourceforge.liferea",
                 _("Liferea unread icon")
             )
             self.indicator.set_status(AppIndicator.IndicatorStatus.ACTIVE)
             self.indicator.set_title("Liferea")
         else:
-            self.read_pix = Liferea.icon_create_from_file("emblem-web.svg")
+            self.read_pix = "net.sourceforge.liferea"
             # FIXME: Support a scalable image!
-            self.unread_pix = Liferea.icon_create_from_file("unread.png")
+            self.unread_pix = "net.sourceforge.liferea"
 
             self.staticon = Gtk.StatusIcon ()
             self.staticon.connect("activate", self.trayicon_click)
@@ -314,7 +316,7 @@
                 # Workaround Mate bug
                 # See also: https://github.com/mate-desktop/mate-panel/issues/1412
                 self.indicator.set_icon_full(
-                    Liferea.icon_find_pixmap_file("unread.png"),
+                    "net.sourceforge.liferea",
                     _("Liferea unread icon")
                 )
 
@@ -324,7 +326,7 @@
                 # Workaround Mate bug
                 # See also: https://github.com/mate-desktop/mate-panel/issues/1412
                 self.indicator.set_icon_full(
-                    Liferea.icon_find_pixmap_file("emblem-web.svg"),
+                    "net.sourceforge.liferea",
                     _("Liferea")
                 )
