From ada4158de9d8cb0127d3d37d495e413e4c39482b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pawe=C5=82=20Marciniak?= <sunwire+git@gmail.com>
Date: Sat, 29 Jan 2022 11:14:41 +0100
Subject: [PATCH 4/4] plugin-installer use XDG_DATA_HOME path when it is set

---
 plugins/plugin-installer.py | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/plugins/plugin-installer.py b/plugins/plugin-installer.py
index 442c9452..74a6bb21 100644
--- a/plugins/plugin-installer.py
+++ b/plugins/plugin-installer.py
@@ -19,6 +19,7 @@
 
 import urllib.request, json
 import os, sys, glob, shutil, subprocess
+from pathlib import Path
 import gi
 
 gi.require_version('Gtk', '3.0')
@@ -64,13 +65,15 @@ class AppActivatable(GObject.Object, Liferea.ShellActivatable):
 
 class PluginBrowser(Gtk.Window):
     SCHEMA_ID = "net.sf.liferea.plugins"
+    SCHEMA_PATH = "glib-2.0/schemas/"
+    PLUGIN_PATH = "liferea/plugins"
 
     def __init__(self):
         Gtk.Window.__init__(self, title=_("Plugin Installer"))
 
-        # FIXME: using safe XDG paths would be better
-        self.target_dir = os.path.expanduser("~/.local/share/liferea/plugins/")
-        self.schema_dir = os.path.expanduser("~/.local/share/glib-2.0/schemas/")
+        self.data_home_path = os.getenv('XDG_DATA_HOME',Path.joinpath(Path.home(), ".local/share"))
+        self.target_dir = Path.joinpath(Path(self.data_home_path), self.PLUGIN_PATH)
+        self.schema_dir = Path.joinpath(Path(self.data_home_path), self.SCHEMA_PATH)
 
         self.set_border_width(10)
         self.set_default_size(600,300)
@@ -91,8 +94,8 @@ class PluginBrowser(Gtk.Window):
             try:
                 name = next(iter(ref))
                 installed = False
-                if(os.path.isfile('%s%s.plugin' % (self.target_dir, ref[name]['module'])) or
-                   os.path.isdir('%s%s' % (self.target_dir, ref[name]['module']))):
+                if(os.path.isfile('%s/%s.plugin' % (self.target_dir, ref[name]['module'])) or
+                   os.path.isdir('%s/%s' % (self.target_dir, ref[name]['module']))):
                    installed = True
                 if not 'icon' in ref[name]:
                    ref[name]['icon'] = 'libpeas-plugin'
-- 
2.35.0

