From 526ab80899e317a5d98743722f675a06e6de0a14 Mon Sep 17 00:00:00 2001
From: bbhtt <62639087+bbhtt@users.noreply.github.com>
Date: Tue, 26 Jul 2022 18:38:21 +0530
Subject: [PATCH] Update release date

---
 net.sourceforge.liferea.appdata.xml.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net.sourceforge.liferea.appdata.xml.in b/net.sourceforge.liferea.appdata.xml.in
index cc9b7cf8..189b7796 100644
--- a/net.sourceforge.liferea.appdata.xml.in
+++ b/net.sourceforge.liferea.appdata.xml.in
@@ -67,6 +67,8 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="1.13.9"  date="2022-07-26" />
+    <release version="1.13.8"  date="2022-04-05" />
     <release version="1.13.7"  date="2021-12-07" />
     <release version="1.13.6"  date="2021-05-07" />
     <release version="1.13.5"  date="2021-01-10" />
-- 
2.37.1

