From 393281dce3b961ed765e1ada100023031fa372d0 Mon Sep 17 00:00:00 2001
From: bbhtt <62639087+bbhtt@users.noreply.github.com>
Date: Thu, 27 Jan 2022 20:02:43 +0530
Subject: [PATCH] Update appdata to 1.13.7

---
 net.sourceforge.liferea.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net.sourceforge.liferea.appdata.xml.in b/net.sourceforge.liferea.appdata.xml.in
index 07f232bb..4d55b731 100644
--- a/net.sourceforge.liferea.appdata.xml.in
+++ b/net.sourceforge.liferea.appdata.xml.in
@@ -67,6 +67,7 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="1.13.7"  date="2021-12-07" />
     <release version="1.13.6"  date="2021-05-07" />
     <release version="1.13.5"  date="2021-01-10" />
     <release version="1.13.4"  date="2020-12-24" />
-- 
2.35.0

