From 2bbcfc3b24e1a6c9c19d08c9cf55533c5aec634d Mon Sep 17 00:00:00 2001
From: Mikel Olasagasti Uranga <mikel@olasagasti.info>
Date: Sun, 9 May 2021 10:50:41 +0200
Subject: [PATCH] Update appdata to 1.13.6

---
 net.sourceforge.liferea.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net.sourceforge.liferea.appdata.xml.in b/net.sourceforge.liferea.appdata.xml.in
index 7e531726..3b23bb91 100644
--- a/net.sourceforge.liferea.appdata.xml.in
+++ b/net.sourceforge.liferea.appdata.xml.in
@@ -68,6 +68,7 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="1.13.6"  date="2021-05-07" />
     <release version="1.13.5"  date="2021-01-10" />
     <release version="1.13.4"  date="2020-12-24" />
     <release version="1.13.3"  date="2020-10-09" />
-- 
2.31.1

