From bb6506ecf55521a51098157d7b545edbcb06569a Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Tue, 21 Mar 2023 06:31:40 +0530
Subject: [PATCH] Update appdata for 1.14.2

---
 net.sourceforge.liferea.appdata.xml.in | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/net.sourceforge.liferea.appdata.xml.in b/net.sourceforge.liferea.appdata.xml.in
index 8fa2c170..8b043edf 100644
--- a/net.sourceforge.liferea.appdata.xml.in
+++ b/net.sourceforge.liferea.appdata.xml.in
@@ -105,6 +105,15 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release date="2023-03-21" version="1.14.2">
+      <description>
+        <p>This is a stability fix for 1.14.1</p>
+        <ul>Changes
+          <li>Fixes #1212: 1.14.1 crash when refreshing feeds (mozbugbox)</li>
+          <li>Fixes a memory leak when reloading CSS (Lars Windolf)</li>
+        </ul>
+      </description>
+    </release>
     <release date="2023-03-13" version="1.14.1">
       <description>
         <p>CVE-2023-1350 Remote code execution on feed enrichment</p>
-- 
2.40.0

